cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -979,17 +983,9 @@ WHERE {
     ?TieFlow
         a cim:TieFlow ;
         cim:TieFlow.Terminal ?terminal ;
-        cim:TieFlow.ControlArea ?ControlArea .
-    ?ControlArea
-        a cim:ControlArea ;
-        cim:IdentifiedObject.name ?controlAreaName ;
-        entsoe:IdentifiedObject.energyIdentCodeEic ?energyIdentCodeEic ;
-        cim:ControlArea.type ?controlAreaType .
-    FILTER (regex(STR(?controlAreaType), "Interchange", "i"))
+        cim:TieFlow.ControlArea ?ControlArea
```

Comment:
Like how information for tie flows and control areas has been separated in the two corresponding queries. Furthermore, now a control area can now be read and created only from EQ data.